dd-java-agent/agent-bootstrap/src/main/java/datadog/trace/bootstrap/instrumentation/decorator/DatabaseClientDecorator.java

```diff
@@ -41,7 +41,10 @@ public AgentSpan onConnection(final AgentSpan span, final CONNECTION connection)
       span.setTag(Tags.DB_INSTANCE, instanceName);
 
       if (instanceName != null && Config.get().isDbClientSplitByInstance()) {
-        span.setServiceName(instanceName);
+        span.setServiceName(
+            Config.get().isDbClientSplitByInstanceTypeSuffix()
+                ? instanceName + "_" + dbType()
```

Comment:
```suggestion
                ? instanceName + "-" + dbType()
```

Please update to a hyphen